src/Formatter/AbstractMarkdownFormatter.php

```diff
@@ -149,14 +149,7 @@ protected function processDescription(string $description): string {
         $in_list = TRUE;
       }
       else {
-        // If previous line was a list item and this is empty - preserve NL.
```

Comment:
Hi @AlexSkrypnyk 
I can not see any cases go into this if statement. 
Please correct me if I was wrong.